react/modules/tasks/class-components.md

```diff
@@ -0,0 +1,60 @@
+# React project setup. Class components. Error boundary.
+
+## Technical Requirements.
```

Comment:
Personally, I don't like those periods at the end

Reply:
Updated